src/components/TaskRunDetails.vue

```diff
@@ -57,28 +65,41 @@
   import { PKeyValue, PTags } from '@prefecthq/prefect-design'
   import DurationIconText from '@/components/DurationIconText.vue'
   import FlowRunIconText from '@/components/FlowRunIconText.vue'
+  import { useTaskRunResult } from '@/compositions'
+  import { localization } from '@/localization'
   import { TaskRun } from '@/models/TaskRun'
   import { formatDateTimeNumeric } from '@/utilities/dates'
   import { secondsToApproximateString } from '@/utilities/seconds'
 
-  defineProps<{
+  const props = defineProps<{
     taskRun: TaskRun,
     alternate?: boolean,
   }>()
+
+  const { result } = useTaskRunResult(props.taskRun.id)
 </script>
 
 <style>
-  .task-run-details {
-    @apply
-    flex
-    flex-col
-    gap-3
-    items-start
-  }
-
-  .task-run-details__tags {
-    @apply
-    mb-1
-    mr-1
-  }
+.task-run-details { @apply
+  flex
+  flex-col
+  gap-3
+  items-start
+}
+
+.task-run-details__tags { @apply
+  mb-1
+  mr-1
+}
+
+.task-run-details__markdown-renderer {
+  font-size: inherit;
+}
+
+/*
+  Note: The core client writes the description as a paragraph token, so we need to override the default margin.
+*/
+.task-run-details__markdown-renderer .markdown-renderer__text--paragraph { @apply
```

Comment:
No way to target a native `<p>` tag or something instead of the mutable nested class selector eh?

Reply:
With that PR I think this can just go away then right?

Reply:
Yup - removed